app/res/layout/component_entity_detail_item.xml

```diff
@@ -48,7 +48,6 @@
             android:layout_marginRight="12dp"
             android:drawableLeft="@drawable/sym_action_call"
             android:drawablePadding="@dimen/content_min_margin"
-            android:inputType="phone"

```

Comment:
This is the core change, right? Does this have meaningful changes outside of removing the one-line limitation? Clicking it still initiates a call, correct?